apps/src/sites/studio/pages/print_certificates/show.js

```diff
@@ -1,6 +1,10 @@
 import analyticsReporter from '@cdo/apps/lib/util/AnalyticsReporter';
 import {EVENTS} from '@cdo/apps/lib/util/AnalyticsConstants';
+import {getStore} from '@cdo/apps/redux';
 
 $(() => {
-  analyticsReporter.sendEvent(EVENTS.CERTIFICATE_PRINT_PAGE_VISITED);
+  const isTeacher = getStore().getState().currentUser.userType === 'teacher';
```

Comment:
Learning Q: is this a good example of using data coming from the database?

Reply:
This is more of an example of getting data from Redux (which, in this case, does come from the database at some point). But, definitely an example of a case where this page will act differently for different users!

Reply:
Cool, ty!